lib/create_command.dart

```diff
@@ -1,25 +1,72 @@
 import 'package:args/args.dart';
+import 'templates/template.dart';
 import 'package:process_run/process_run.dart';
 
+import 'utils.dart';
+
 void createCommand(ArgResults command) async {
-  final org = command['org'] as String;
-  if (org == null || org.isEmpty) {
-    print('The org option is required.');
-    return;
-  }
-  final name = command['name'] as String;
-  if (name == null || name.isEmpty) {
-    print('The name option is required.');
-    return;
-  }
+  final interactive = command['interactive'] != 'false';
+
+  final name = getString(
+    command,
+    interactive,
+    'name',
+    'Choose a name for your project: ',
+    desc: 'Note: this must be a valid dart identifier (no dashes).',
+  );
+
+  final org = getString(
+    command,
+    interactive,
+    'org',
+    'Choose an org for your project: ',
+    desc:
+        'Note: this is a dot separated list of "packages", normally in reverse domain notation.',
+  );
+
+  final currentDir =
```

Comment:
Can't you use `Directory.current` to get the current working directory? That would be more cross platform and you won't have to parse the stdout response.